### PR TITLE
SEED-729: Profile Join Template 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review

**Comment by peterp** [comment]
@Mitso Did you forgot to commit the HTML?

File: iogt/static/css/modules/_m-languages.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 }
 .current-language, .select-language {
+  color: #ffffff;
```
Comment (peterp) [inline iogt/static/css/modules/_m-languages.scss]: `#fff` should be fine.

**Comment by Mitso** [comment]
@peterp the above is updated :), please review.

File: iogt/static/css/modules/_m-heading.scss [hunk 2]
```diff
@@ -1,12 +1,20 @@
 
 .heading {
   background: #eee;
+  padding: 0 0 15px;
 }
 
 .heading--large {
   font-size: 32px;
```
Comment (peterp) [inline iogt/static/css/modules/_m-heading.scss:8]: Should use variables.

File: iogt/static/css/modules/_m-profiles.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+.profiles {
+  background: #fff;
```
Comment (peterp) [inline iogt/static/css/modules/_m-profiles.scss:3]: is this required? isn't the background white by default?

File: iogt/static/css/modules/_m-search.scss [hunk 4]
```diff
@@ -9,8 +9,8 @@
   padding: 15px;
 
   input[type="submit"] {
```
Comment (peterp) [inline iogt/static/css/modules/_m-search.scss:11]: These look like smart phone styles?

**Pull request merged**